### ISSUE TITLE
MacOS docs?

### ISSUE THREAD

**Issue by jamesgecko** [MicrosoftDocs/BingAds-docs#317]
This is hard to follow on MacOS using the dotnet command line tool and Visual Studio Code.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d2706bd8-f868-05b8-ab4b-9440628296c9
* Version Independent ID: 6d0243d6-024a-5c3c-4b70-2c56db079b5c
* Content: [Walkthrough: Bing Ads Web Application in C# - Bing Ads](https://docs.microsoft.com/en-us/bingads/guides/walkthrough-web-application-csharp?view=bingads-12)
* Content Source: [BingAds/bingads-12/guides/walkthrough-web-application-csharp.md](https://github.com/MicrosoftDocs/BingAds-docs/blob/live/BingAds/bingads-12/guides/walkthrough-web-application-csharp.md)
* Service: **bing-ads**
* GitHub Login: @eric-urban
* Microsoft Alias: **eur**

**Comment by eric-urban** [comment]
@jamesgecko since you mentioned the console, I suggest that you try [this walkthrough](https://docs.microsoft.com/en-us/bingads/guides/walkthrough-desktop-application-csharp?view=bingads-12) instead. Whereas the walkthroughs are not available for each combination of platforms and tool sets, please feel free to ask detailed questions via the [forum](https://social.msdn.microsoft.com/Forums/en-US/home?forum=BingAds) with repro steps if applicable, and we'll be happy to assist.
